tests/Schema/MigratorTest.php

```diff
@@ -231,7 +234,11 @@ public function testCharacterTypeFieldLong(string $type, bool $isBinary, int $le
 
         // functional test for Expression::escapeStringLiteral() method
         $strRaw = $model->getPersistence()->typecastSaveField($model->getField('v'), $str);
-        $strRawSql = \Closure::bind(static fn () => $model->expr('')->escapeStringLiteral($strRaw), null, Expression::class)();
+        $strRawSql = \Closure::bind(
```

Comment:
We would have to do this CS change consistently across all repos which is not easily possible, as `\Closure::bind` is very often used inside some expr. (like `if`)

please revert these CS changes - if they were done by your IDE, I advise you to disable them for atk4/* repos at least